### PR TITLE
chore(docs): replace getStaticProps with static rendering in app deploy documentation

### PR DESCRIPTION
**Pull request by AntoineBourin**
### What?

- Deploy documentation in `App router` is talking about `getStaticProps` function. As this function doesn't exist anymore in this router version, we can replace it with the Static Rendering link.

### Why?

- `getStaticProps` function doesn't exist in the `App Router` context.

### How?

- See doc : `getStaticProps` replaced by link to Static Rendering documentation to let users see how to build static pages.

### CODE REVIEW

**Comment by AntoineBourin** [comment]
Maybe we can include `using generateStaticParams for dynamic segments` ?

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 082fecdae545e4774a7b92997d71c843f62144ba

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by styfle — CHANGES_REQUESTED** [review]
The comment at the top indicates this document is shared:

> The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component.

So we need two different links: one with `<PagesOnly>Content</PagesOnly>` and one with `<AppOnly>Content</AppOnly>`

**Comment by AntoineBourin** [comment]
@styfle Thanks ! Did not see the comment ! 
Documentation updated.

**Review by styfle — DISMISSED** [review]
I'm not sure if this will work since its splitting up the `<li>` instead of wrapping the `<ul>`.

We can try it and revert if it causes a problem.

**Review by timneutkens — DISMISSED** [review]
Looks good! Thanks!

**Review by timneutkens — CHANGES_REQUESTED** [review]
Seems like the linting messes up the formatting here. Could you have a look at that?

File: docs/02-app/01-building-your-application/07-deploying/index.mdx [hunk 1]
```diff
@@ -11,10 +11,16 @@ Congratulations! You're here because you are ready to deploy your Next.js applic
 
 `next build` generates an optimized version of your application for production. This standard output includes:
 
-- HTML files for pages using `getStaticProps` or [Automatic Static Optimization](/docs/pages/building-your-application/rendering/automatic-static-optimization)
```
Comment (leerob) [inline docs/02-app/01-building-your-application/07-deploying/index.mdx:14]: I'd say let's just make this more generic for both:

- HTML files for static pages

**Comment by leerob** [comment]
Closing as stale from https://github.com/vercel/next.js/pull/53039#discussion_r1303677101, but open to a new PR 🙏

**Pull request closed without merging**